src/entities/DFGVariable.jl

```diff
@@ -122,7 +138,12 @@ solverData(v::DFGVariable, key::Symbol=:default) = haskey(v.solverDataDict, key)
 Retrieve data structure stored in a variable.
 """
 function getData(v::DFGVariable; solveKey::Symbol=:default)::VariableNodeData
-  @warn "getData is deprecated, please use solverData()"
+  #FIXME but back in later, it just slows everything down
+  if !(@isdefined getDataWarnOnce)
+    @warn "getData is deprecated, please use solverData(), future warnings in getData is suppressed"
+    global getDataWarnOnce = true
+  end
+  # @warn "getData is deprecated, please use solverData()"
```

Comment:
Got tired of messages, we can remove this since IIF was updated.

Reply:
funny story, the remaining message now all come from within DFG

Reply:
It should be fixed now

Reply:
thanks for the fix!